js/notebook.js

```diff
@@ -1,19 +1,36 @@
+// Haven't decided what to do with this yet
+
 function addNotebook() {
-    var noteDiv = document.getElementById("nav_notebooks");
+    const noteDiv = document.getElementById("nav_notebooks");
     noteDiv.append(createNewNotebook());
     return false;
 }
 
 function createNewNotebook() {
-    // Create the structure of the Notebook, which is just an <a>
-    var a = document.createElement("a");
+    // Create the structure of the Notebook, which is an <input>
+    const input = document.createElement("input");
+    input.setAttribute("type", "text");
+    input.setAttribute("readOnly", "true");
+
+    // Set up the new <input> classes then add the text
+    input.className = "list-group-item list-group-item-action";
+    input.id = "notebook";
+    input.setAttribute("value", "New Notebook");
+    input.setAttribute("onclick", "showNotebook()");
+    input.setAttribute("ondblclick", "renameNotebook()");
 
-    // Set up the new <a> classes then add the text
-    a.className = "list-group-item list-group-item-action";
-    a.id = "notebook";
-    // a.onclick = "showNotebook()"; -> Displays notebook contents in right-side display
-    a.append(document.createTextNode("New Notebook"));
+    // Return the <input>
+    return input;
+}
 
-    // Return the <a>
-    return a;
+function renameNotebook() {
+    // Toggle editable for element
+    console.log("dblclick works")
```

Comment:
If this is just for debugging, please remove it in the final code!

Reply:
Notebook.js is superficial to the current code. It's connected to a previous version I was working on, but I haven't decided if I want to implement both versions or just one, so I left it as it is. 
I'll remove the debugging portions once it's been merged though. I forgot I left them there.